src/auth/mod.rs

```diff
@@ -68,6 +68,3 @@ pub use authenticator::{AuthenticationError, Authenticator};
 
 mod user;
 pub use user::{DefaultUser, UserDetail};
-
-#[cfg(all(feature = "pam_auth", target_family = "unix"))]
```

Comment:
question(non-blocking): why is this not need if it is a separate crate? Shouldn't we prevent it to be built on systems without `pam`?

Reply:
The module is completely removed here from libunftp, thats why its removed here. But I think your other comment is valid too. I looked at [the pam crate](https://github.com/1wilkens/pam/) to try and see how they do it and I don't see any target_family = "unix" stuff. These will now move to unFTP itself.